client/state/ui/test/reducer.js

```diff
@@ -29,7 +29,8 @@ describe( 'reducer', () => {
 			'olark',
 			'preview',
 			'actionLog',
-			'happychat'
+			'happychat',
+			'mediaModal'
```

Comment:
Yeah, I'm not a fan of the trailing commas. Looks untidy to me, and I'd argue could be error-prone in their own regard if we rely on them too much (I don't think people are putting trailing commas on hugely nested objects? Seems inconsistent anyways).

Anywho, I plead personal preference 😄